code/C1-files.py

```diff
@@ -14,7 +14,7 @@
 ##########
 
 gateway_hostname  = 'faraday.inria.fr'
-gateway_username  = 'inria_r2lab.tutorial'
+gateway_username  = 'root'
```

Comment:
what ? 
we cannot ask or even suggest the average user to use the root account to log onto faraday !
please explain..

Reply:
This is a remnant of when I tested the tutorials. This is now corrected.